e2e-tests/portal/cypress/fixtures/users.js

```diff
@@ -1,33 +1,15 @@
-const MOCK_USERS = require('../../../../utils/mock-data-loader/portal/users');
-const MOCK_TFM_USERS = require('../../../../utils/mock-data-loader/tfm/mocks/users');
-const { USER_ROLES } = require('./constants');
-
-const BANK1_MAKER1 = MOCK_USERS.find((user) => user.roles.includes(USER_ROLES.MAKER) && user.username === 'BANK1_MAKER1');
-
-const BANK1_MAKER2 = MOCK_USERS.find((user) => user.roles.includes(USER_ROLES.MAKER) && user.username === 'BANK1_MAKER2');
-
-const BANK2_MAKER2 = MOCK_USERS.find((user) => user.roles.includes(USER_ROLES.MAKER) && user.username === 'BANK2_MAKER2');
-
-const BANK3_GEF_MAKER1 = MOCK_USERS.find((user) => user.roles.includes(USER_ROLES.MAKER) && user.username === 'BANK3_GEF_MAKER1');
-
-const BANK1_CHECKER1 = MOCK_USERS.find((user) => user.roles.includes(USER_ROLES.CHECKER) && user.username === 'BANK1_CHECKER1');
-
-const BANK1_READ_ONLY1 = MOCK_USERS.find((user) => user.roles.includes(USER_ROLES.READ_ONLY) && user.username === 'BANK1_READ_ONLY1');
-
-const ADMIN = MOCK_USERS.find((user) => user.username === 'ADMIN');
-
-// TFM
-const UNDERWRITER_MANAGER = MOCK_TFM_USERS.find((user) => user.teams.includes('UNDERWRITER_MANAGERS'));
-
-const USER_WITH_INJECTION = {
-  username: '{ "$gt": "" }',
-  email: '{ "$gt": "" }',
-  password: 'TestPassword123!',
-  firstname: 'test',
-  surname: 'injection',
-  bank: 'HSBC',
-  roles: [USER_ROLES.MAKER],
-};
+const {
+  BANK1_MAKER1,
+  BANK1_MAKER2,
+  BANK2_MAKER2,
+  BANK3_GEF_MAKER1,
+  BANK1_CHECKER1,
+  BANK1_READ_ONLY1,
+  ADMIN,
+  READ_ONLY_ALL_BANKS,
+  UNDERWRITER_MANAGER,
+  USER_WITH_INJECTION,
+} = require('../../../e2e-fixtures/users');
```

Comment:
Is there a reason that we're even keeping this file?

Reply:
I can delete it now, did this as an intermediary so I wasn't changing 100s of files

Reply:
I think it would be nice -- Im not sure it serves any purpose (if the other services dont have this file -- itll be bringing it in line)

Reply:
This file is deleted now